### ISSUE TITLE
SnappyDB with SQLiteDB

### ISSUE THREAD

**Issue by DeepakSenapati** [nhachicha/SnappyDB#19]
First of all I'm very impressed with this library's ease of use and performance. In my app, I'm using another library which is very much required but it uses SQLiteDB internally. Now after integrating this with my present app, which has SnappyDB, I get an error - 

java.lang.UnsatisfiedLinkError: Couldn't load snappydb-native from loader dalvik.system.PathClassLoader[dexPath=/data/app/com.package.name-84.apk,libraryPath=/data/app-lib/com.package.name-84]: findLibrary returned null

Is there any way I can integrate both snappyDB abd SQLiteDB in my app and make both work.

**Comment by thiyagab** [comment]
Hi we got the same issue, when running snappydb with 64 bit processor, please check the below details,
lib/arm64-v8a/libpl_droidsonroids_gif.so
lib/armeabi/libpl_droidsonroids_gif.so
lib/armeabi-v7a/libpl_droidsonroids_gif.so
lib/mips/libpl_droidsonroids_gif.so
lib/mips64/libpl_droidsonroids_gif.so
lib/x86/libpl_droidsonroids_gif.so
lib/x86_64/libpl_droidsonroids_gif.so
lib/armeabi/libsnappydb-native.so
lib/armeabi-v7a/libsnappydb-native.so
lib/mips/libsnappydb-native.so
lib/x86/libsnappydb-native.so

**Comment by nhachicha** [comment]
Hi @thiyagab 
please check https://github.com/nhachicha/SnappyDB/issues/45 this should be fixed by using the new snapshot build that include the correct AAR with the 64bit files.

```groovy
repositories {
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}

dependencies {
    compile 'com.snappydb:snappydb-lib:0.5.2-SNAPSHOT'
    compile 'com.esotericsoftware.kryo:kryo:2.24.0'
}
````

Cheers

**Comment by thiyagab** [comment]
Worked perfectly Thanks

**Comment by nhachicha** [comment]
Fixed in `0.5.2`
